app/views/api/integrations/apicast/shared/_policies.html.slim

```diff
@@ -1,13 +1,18 @@
-- content_for :javascripts do
-  = javascript_pack_tag 'policies'
-
-= f.toggled_inputs 'policies', cookie_name: 'policies-rules' , legend: "Policies", id: "policies-container" do
+= f.toggled_inputs 'policies', cookie_name: 'policies-rules', legend: "Policies", id: "policies-container" do
   div#policies
 
-javascript:
-  document.addEventListener('DOMContentLoaded', function () {
-    var registry = #{json Policies::PoliciesListService.call(current_account)}
-    var chain = #{json @proxy.policies_config}
-    var serviceId = #{json @service.id}
-    initPolicies({element: 'policies', registry: registry, chain: chain, serviceId: serviceId})
-  })
+  - if @error.nil?
+    - content_for :javascripts do
+      = javascript_pack_tag 'policies'
+
+    javascript:
+      document.addEventListener('DOMContentLoaded', function () {
```

Comment:
So I thought we query the policies through an AJAX request ... I think we should but probably out of the scope of this PR

@3scale/system some wdyt?

Reply:
I agree with Hery but I think it should be out of the scope of the PR. And eveb better if it is done after https://github.com/3scale/porta/pull/826/files#discussion_r289074548